fat.py

```diff
@@ -144,6 +161,8 @@ def main():
     parser = argparse.ArgumentParser()
     parser.add_argument("firm_path", help="The path to the firmware image", type=str)
     parser.add_argument("-q", "--qemu", metavar="qemu_path", help="The qemu version to use (must exist within qemu-builds directory). If not specified, the qemu version installed system-wide will be used", type=str)
+    parser.add_argument("-s","--sql", dest="sql", action="store", default=None, help="Hostname of SQL server")
+    parser.add_argument("-b","--brand", dest="brand", action="store", default=None, help="Brand of the firmware image")
```

Comment:
We are not using PostgreSQL database to store information about the emulated images.

From README,

> Firmware Analysis Toolkit (FAT henceforth) is based on Firmadyne with some changes. Firmadyne uses a PostgreSQL database to store information about the emulated images. However just for the core functionality i.e. emulating firmware, PostgreSQL is not really needed. Hence FAT doesn't use it.

Reply:
Yes, FAT does not need it. I guess it won't harm to pass those parameter to firmadyne so user can use FAT even if they need the firmadyne sql feature/need to do some manual task with firmadyne later.